.gitignore

```diff
@@ -11,8 +11,14 @@
 tags
 Module.symvers
 Module.markers
+*.mod
 *.mod.c
 modules.order
 __pycache__/
 *.py[cod]
 *$py.class
+*.out
+
+casadm/casadm
+modules/include/ocf
```

Comment:
You may want to include `modules/cas_cache/src/ocf` as well.

Reply:
cas_cache has its own .gitignore